attribute-service/src/main/resources/configs/common/application.conf

```diff
@@ -3,11 +3,17 @@ service.name = attribute-service
 service.admin.port = 9013
 document.store {
   dataStoreType = mongo
+  appName = attribute-service-local
   mongo {
-    host = localhost
-    host = ${?MONGO_HOST} # provides a way to override the mongo_host via an environment variable
-    port = 27017
-    port = ${?MONGO_PORT}
+    database = default_db
+    endpoints = [
+      {
+        host = localhost
+        host = ${?MONGO_HOST} # provides a way to override the mongo_host via an environment variable
+        port = 27017
+        port = ${?MONGO_PORT}
+      }
```

Comment:
Not consuming the mounted credentials (env. vars.)?

Reply:
This was intended, but I could be convinced to change it.

My goal was basically to expose a way to use credentials without requiring them which would be a breaking change for anyone currently using attribute service. Basically keep the same default behavior while extending the flexibility of the config.  If we can map the default to env vars and it ignores them gracefully if env not set, then I'll go back and add em in - let me check that.

Reply:
tested this, they're ignored if the env var isn't set so it works correctly. Added them in